shared/oae/css/oae.base.css

```diff
@@ -578,6 +627,13 @@ ul.as-selections li.as-selection-item.as-ghost-item.as-ghost-selected a.as-close
     display: block;
 }
 
+/* No results message */
+.oae-no-empty-text li.as-message {
```

Comment:
Removed. I think it was added as part of an attempt to stop the results from showing if there were no items, but I wound up identifying a bug in auto-suggest instead and fixing it (documented in autosuggest changes now)